doc/build-windows.md

```diff
@@ -84,6 +85,7 @@ Then build using:
 
     cd depends
     make HOST=i686-w64-mingw32
+    
```

Comment:
That would be repeating the same thing twice, maybe it would be better to add a *warning* section describing the problem, instead of buried in the instructions to type? (and possibly refer to that here)